stratum/hal/bin/barefoot/deb/start-stratum.sh

```diff
@@ -56,7 +56,7 @@ fi
 
 mkdir -p /var/run/stratum /var/log/stratum
 
-exec /usr/bin/stratum_bf \
+exec /usr/bin/stratum_bfrt \
```

Comment:
This will break the bf target

Reply:
What about adding the following to stratum_binaries:

```
    symlinks = { "/usr/bin/stratum_bf": "/usr/bin/stratum_bfrt" },
```

Reply:
Great!


```
dpkg --contents bazel-bin/stratum/hal/bin/barefoot/stratum-bfrt_0.0.1_amd64.deb
drwxr-xr-x 0/0               0 2000-01-01 00:00 ./
drwxr-xr-x 0/0               0 2000-01-01 00:00 ./usr/
drwxr-xr-x 0/0               0 2000-01-01 00:00 ./usr/bin/
-rwxr-xr-x 0/0            1841 2000-01-01 00:00 ./usr/bin/start-stratum.sh
-rwxr-xr-x 0/0        18031216 2000-01-01 00:00 ./usr/bin/stratum_bfrt
lrwxr-xr-x 0/0               0 2000-01-01 00:00 ./usr/bin/stratum_bf -> ./usr/bin/stratum_bfrt
```

Reply:
A nice side effect is that users who started Stratum with their own script won't have to change those either.